### ISSUE TITLE
Stay logged in and remember language choice 

### ISSUE THREAD

**Issue by pogster** [raithos/xwing#231]
### Expected Behavior

1. I open yasb 2.0.
2. It's in german but I want it to be english so I change the setting
3. I log in with Google to save and load squads
4. I close the Browser (tab) 
5. I return at a later time
6. I expect to be still logged in and the language to be set to english

### Actual Behavior

1. I return at a later time
2. I'm logged out and language is returned to german. 

### Language

German/english

**Comment by raithos** [comment]
I wasn't able to replicate this error. Are your cookies enabled for the site? the language settings are stored in cookies.

**Comment by pogster** [comment]
I did some testing and it seems to remember me as long as I don't close the browser.
I'm not aware of disabling cookies in general or for the site or changing any settings for that in general (especially not in Firefox which I nearly don't use at all).

I would have the expectation that it remembers me between browser sessions for at least a month or something?

Reproduced on:
Firefox 63.0.3 (64-bit)
Chrome Version 60.0.3112.78 (Official Build) (64-bit)
